provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/task/RegisterTask.java

```diff
@@ -101,7 +101,8 @@ public void run(ResponseData responseData, Object context) throws ProvisioningDe
             throw new ProvisioningDeviceClientException(new IllegalArgumentException("authorization cannot be null"));
         }
 
-        //SRS_RegisterTask_25_001: [ Constructor shall save provisioningDeviceClientConfig , securityProvider, provisioningDeviceClientContract and authorization.]
+        // check if using operational CSR
+        this.operationalClientCertificateRequest = provisioningDeviceClientConfig.getOperationalCertificateRequest();
```

Comment:
The comment above references some checking happening here, but the code doesn't have any checks in this part of the implementation. Is that pending, or is the comment out-of-date?